MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -89,9 +89,10 @@ private static void ProcessItemCommand(GameSession session, string command)
                 Uid = Environment.TickCount64,
                 CreationTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds(),
                 TransferFlag = TransferFlag.Splitable | TransferFlag.Tradeable,
-                Stats = stats
+                Stats = stats,
+                PlayCount = itemId.ToString().StartsWith("35") ? 10 : 0
             };
-            int.TryParse(config.GetValueOrDefault("rarity", "5"), out item.Rarity);
+            // int.TryParse(config.GetValueOrDefault("rarity", "5"), out item.Rarity);
```

Comment:
Any reason to this be commented?

Reply:
Yeah the rarity is parsed from item data now so we don't need the command, but I left it there in case somebody wanted to spawn an item with a different rarity.